tests/backstop/Dockerfile

```diff
@@ -1,33 +1,38 @@
-FROM node:22.14.0-bookworm
+FROM node:24.2.0-bookworm
 
-# Prevent intractive prompts from apt-get.
+# Prevent interactive prompts from apt-get.
 ARG DEBIAN_FRONTEND=noninteractive
 
 ARG BACKSTOPJS_VERSION=6.3.25
 ENV BACKSTOPJS_VERSION=$BACKSTOPJS_VERSION
 
-RUN apt-get update
-
-# Install suggested requirements for running backstop in GitHub actions.
-# https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-on-gitlabci
-RUN apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 \
+# This version is the only version available on both arm64 and amd64 architectures as of June 2025.
+ARG CHROMIUM_VERSION=135.0.7049.95-1~deb12u1
```

Comment:
As mentioned in [this comment](https://github.com/google/site-kit-wp/issues/10791#issuecomment-2966146910) Debian has changed the package repository underneath us and this version is now the only version of both chromium and chromium-common that has a package for both arm64 and amd64 architectures.